<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="currentValveStatus">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Valve Position</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QComboBox" name="desiredValvePosition">
      <item>
       <property name="text">
        <string>Position 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Position 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Position 3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Position 4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Position 5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Position 6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Position 7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Position 8</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QComboBox" name="desiredRotationDirection">
      <item>
       <property name="text">
        <string>Clockwise</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Counter Clockwise</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>New Valve Position</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Direction</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="changeValvePosition">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Change Position</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
    <addaction name="actionInitializeHamilton"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionInitializeHamilton">
   <property name="text">
    <string>Initialize Hamilton</string>
   </property>
  </action>
  <action name="actionReaddressHamilton">
   <property name="text">
    <string>Readdress Hamilton</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
